test/e2e-wi-tests.sh

```diff
@@ -137,6 +138,8 @@ function gcp_broker_setup() {
   fi
   kubectl annotate --overwrite serviceaccount ${BROKER_SERVICE_ACCOUNT} iam.gke.io/gcp-service-account="${PUBSUB_SERVICE_ACCOUNT_EMAIL}" \
     --namespace "${CONTROL_PLANE_NAMESPACE}"
+  # Add a default broker to trigger brokercell ingress/fanout/retry components.
+  ko apply -f ${CONFIG_GCP_BROKER}
```

Comment:
It seems the intention is to use this broker to "warm up" the data plane? Maybe add a note to explain why you need this. And maybe rename the yaml to `warmup-broker.yaml`.

Reply:
Done. I open a new PR which includes this change  https://github.com/google/knative-gcp/pull/1415, it is a little bit messy in this PR, so I will close it.